### PR TITLE
Imported OmegaT package

### PR DESCRIPTION
**Pull request by jacobzimmermann**
### Please confirm your submission meets all the criteria

- [X ] I have read the [App Requirements][reqs] and [App Maintenance][maint] pages.
- [X] My pull request follows the instructions at [App Submission][submission].
- [X] I am using only the minimal set of permissions. *(If not, please explain each non-standard permission.)*
- [X ] All assets referenced in the manifest are redistributable by any party.  If not, the unredistributable parts are using an extra-data source type.
- [ X] I am an upstream contributor to the project. If not, I contacted upstream developers about submitting their software to Flathub. **Link: https://omegat.org**
- [ ] I own the domain used in the application ID or the domain has a policy for delegating subdomains (e.g. GitHub, SourceForge). No: I do not control the project or its domain name, however I did contact them about releasing a flatpak package and they agree to that.
- [X] Any additional patches or files have been submitted to the upstream projects concerned. *(If not, explain why.)* N/A

[reqs]: https://github.com/flathub/flathub/wiki/App-Requirements
[maint]: https://github.com/flathub/flathub/wiki/App-Maintenance
[submission]: https://github.com/flathub/flathub/wiki/App-Submission

### CODE REVIEW

File: org.omegat.OmegaT.yml [hunk 1]
```diff
@@ -0,0 +1,69 @@
...
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+command: run.sh
+finish-args:
+  - --filesystem=xdg-documents
+  - --socket=wayland
+  - --socket=x11
```
Comment (hfiguiere) [inline org.omegat.OmegaT.yml:12]: 1. x11 need `--share=ipc`
2. it should be `--socket=fallback-x11` if `--socket=wayland` is present.

**Comment by hfiguiere** [comment]
bot, build org.omegat.OmegaT

File: org.omegat.OmegaT.appdata.xml [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+	<id>org.omegat.OmegaT</id>
+	<name>OmegaT</name>
+	<summary>A simple, open-source translation memory manager and CAT tool</summary>
+	<metadata_license>CC0-1.0</metadata_license>
+	<description>
+		<p>OmegaT is a free translation memory application that works on Windows, macOS, Linux... It is a tool intended for professional translators. It does not translate for you! (Software that does this is called "machine translation", and you will have to look elsewhere for it.)</p>
+	</description>
+	<url type="homepage">https://omegat.org/</url>
+	<project_license>GPL-3.0</project_license>
+	<developer_name>The OmegaT Project</developer_name>
```
Comment (hfiguiere) [inline org.omegat.OmegaT.appdata.xml:13]: it's missing oars rating. See https://hughsie.github.io/oars/ 

(the build will fail for this)

**Review by hfiguiere — CHANGES_REQUESTED** [review]

**Comment by hfiguiere** [comment]
The appstream file is also missing screenshots.

**Comment by jacobzimmermann** [comment]
Pushed a new commit addressing those issues.

**Comment by hfiguiere** [comment]
bot, build org.omegat.OmegaT

File: org.omegat.OmegaT.appdata.xml [hunk 3]
```diff
@@ -0,0 +1,31 @@
...
+	<releases>
+		<release date="2021-04-07" version="5.4.3" />
+	</releases>
+	
+	<screenshots>
+		<screenshot type="default">
+			<image>https://omegat.org/img/screenshots/main_window_en.png</image>
+		</screenshot>
+		<screenshot>
+			<image>file://omegat-screenshot.png</image>
```
Comment (hfiguiere) [inline org.omegat.OmegaT.appdata.xml:28]: `file:` URL are not allowed in appstream (on flathub)

That's why the build failed.

**Comment by jacobzimmermann** [comment]
Removed the `file:` URL.

**Comment by hfiguiere** [comment]
bot, build org.omegat.OmegaT

File: org.omegat.OmegaT.yml [hunk 4]
```diff
@@ -0,0 +1,69 @@
...
+  - share/include
+  - share/cmake*
+  - share/autoconf
+  - share/automake*
+  - share/bash-competions
+  - share/doc
+  - share/man
+  - share/vala
+  - share/info
+  - usr/bin/x86_64-unknown*
```
Comment (hfiguiere) [inline org.omegat.OmegaT.yml:26]: is there a reason this is in the cleanup section? I would think the executable binaries would need to stay in the flatpak.

Comment (jacobzimmermann) [reply org.omegat.OmegaT.yml:26]: These are not the application's binaries but part of the buildchain (C, C++ and Vala compilers) which get pulled in during the build. They are not needed to run the app.

File: org.omegat.OmegaT.yml (hunk 1 above)
Comment (barthalion) [reply org.omegat.OmegaT.yml:11]: I don't think Java apps talk Wayland.

Comment (jacobzimmermann) [reply org.omegat.OmegaT.yml:11]: That has been fixed in the current commit

**Comment by barthalion** [comment]
Have you contacted upstream about this submission?

File: org.omegat.OmegaT.yml (hunk 4 above)
Comment (barthalion) [reply org.omegat.OmegaT.yml:26]: I don't see any of these files in the built app even though I removed `cleanup` section locally.

**Comment by jacobzimmermann** [comment]
> Have you contacted upstream about this submission?

Yes, they approve of it.

**Comment by barthalion** [comment]
In that case, are they interested in co-maintaining it on Flathub? Just so they have an access in case they would like to do some changes on their own.

**Comment by jacobzimmermann** [comment]
> In that case, are they interested in co-maintaining it on Flathub? Just so they have an access in case they would like to do some changes on their own.

They say at the moment they don't have the time or manpower to maintain a Flatpak package. If the package is published under their namespace (i.e. org.omegat.OmegaT) they want to leave the possibility open for them to reclaim the package and listing in the future if the situation arises, or, if I don't agree to that, then they say I should publish it under my own namespace. I'm more than happy to hand it over to them if and when they ask for it, and also I believe that publishing it under another namespace would be confusing for users.

File: org.omegat.OmegaT.yml [hunk 5]
```diff
@@ -0,0 +1,70 @@
...
```
Comment (barthalion) [inline org.omegat.OmegaT.yml:27]:
```suggestion
```

**Pull request closed without merging**